integration/clean/clean_ami.go

```diff
@@ -0,0 +1,85 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+// +build clean
+
+package main
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/aws"
```

Comment:
Any particular reason we're taking a new dependency on v2? At least, it looks new based on the go mod changes

Reply:
Just to make it future proof. I spoke with @jhnlsn and he said we do not have plans to upgrade to v2 int he near future. The aws sdk still gets security updates.